src/MPS.jl

```diff
@@ -583,37 +582,41 @@ function evolve!(::NonCanonical, ψ::AbstractAnsatz, mpo::AbstractMPO; threshold
         groupinds!(ψ, right_inds[i])
     end
 
-    if !isnothing(threshold) || !isnothing(maxdim)
-        truncate_sweep!(form(ψ), ψ; threshold, maxdim, normalize)
+    if any(!isnothing, get.(Ref(kwargs), [:threshold, :maxdim], nothing))
+        truncate_sweep!(form(ψ), ψ; kwargs...)
     else
+        normalize = get(kwargs, :normalize, true)
         normalize && normalize!(ψ)
     end
 
     return ψ
 end
 
-function evolve!(::MixedCanonical, ψ::AbstractAnsatz, mpo::AbstractMPO; normalize, kwargs...)
+function evolve!(::MixedCanonical, ψ::AbstractAnsatz, mpo::AbstractMPO; kwargs...)
     initial_form = form(ψ)
     mixed_canonize!(ψ, Site(nsites(ψ))) # We convert all the tensors to left-canonical form
 
+    normalize = get(kwargs, :normalize, false)
```

Comment:
here the default is `false` but `NonCanonical` is `true`?
we must keep consistent